### DISCUSSION TITLE
Only see logo of Home Assistant in stead of SVG-image and 'Cannot find element in SVG file'

### DISCUSSION THREAD

**Discussion by rvisser88** [ExperienceLovelace/ha-floorplan#319] [Q&A] [answered]
Goodday,

I'am new with floorplan, but work my way true.
Now i'am stuck at the point where i want to see my SVG in Home Assistant.
First it gave an error that it cannot find 'local/...' so i changed it to 'config/...' and now that error is fixed.
Now it gave the error 'WARNING CONFIG cannot find element ..... in SVG file. 
(https://drive.google.com/file/d/1KCCKJ31KnNmE78Z7i9GeCyxSf85rRm7n/view?usp=share_link)
What can be the problem?

Thanks in advance!!!
Have a nice day,
Rick

**Comment by OzGav** [comment, marked as answer]
You don’t show your yaml which would be helpful. Regardless, type the url you have in your yaml for your SVG into the browser address bar and see if it appears. If it doesn’t you have the wrong path to it.

**Reply by OzGav** [reply]
The error you are now seeing indicates that you have an entity or element in your yaml (e.g. light.hue1) but that ID has not been assigned to any object in your SVG (or, as may be the case, the SVG can’t be found)

**Reply by OzGav** [reply]
Also looking at what you wrote above, the path to your SVG should have been `/local/…` not `local/…`

**Reply by rvisser88** [reply]
The problem was that it couldnt find it locally.
I had to use /api in stead of /local.
How can i set that can use /local to?

**Reply by OzGav** [reply]
Where have you actually put your SVG? A screen capture showing the path would help

**Reply by OzGav** [reply]
Looking at your original response you should enter `http://homeassistant.local:8123/local/floorplan/home_simple/home_simple.svg`

If the above works then in your yaml the image location should be `/local/floorplan/home_simple/home_simple.svg`

**Reply by rvisser88** [reply]
screencapture: https://drive.google.com/file/d/1fK_5H2ed10PNL2CLw3ESDAkMrS3T9GMZ/view?usp=share_link

the only links that work starts with: http://homeassistant.local:8123/api/hassio_ingress/

**Reply by OzGav** [reply]
Ok thanks. Please post your yaml and highlight it all and press the <> button so we can see the formatting

**Reply by OzGav** [reply]
One thing I do notice is in your image set section, the service data url is incorrect as it has an extra `home_simple/`

**Reply by exetico** [reply]
I''l mark this as the answer right now. Let us know if you're still facing issues, rvisser88 :)

**Reply by GngrClapz** [reply]
im having this same problem and so far nothing has fixed it, originally i dont get any picture unless i change   `  image: /local/floorplan/home_simple/home_simple.svg` stylesheet: /local/floorplan/home_simple/home_simple.css ` service_data: /local/floorplan/home_simple/light_${entity.state}.svg` all three to /cofig/local/floorplan etc then the only picture i get is a large HA logo like him  if i use the exact same code from the video i get nothing but 2 errors  ( 
![Screenshot 2023-05-16 120401](https://github.com/ExperienceLovelace/ha-floorplan/assets/133083982/3e194194-de49-4827-bcc9-9b275b825696)
)

**Reply by exetico** [reply]
⬆️ 
GngrClapz created a discussion here:
https://github.com/ExperienceLovelace/ha-floorplan/discussions/336#discussioncomment-5919567

(Just for other people hitting this by searching here, or coming from Google 😄 )
